bundle/manifests/oadp-operator.oadp-cli.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: oadp-cli
+spec:
+  description: oadp-cli is the command line tool for working with the OADP operator
+  displayName: oadp - OADP operator Command Line Interface (CLI)
+  links:
+    - href: https://github.com/migtools/oadp-cli/releases/
```

Comment:
for now this is fine, but the cli needs to be built by Red Hat.. so we'll need a container in openshift-adp hosting the built artifcats.

Reply:
Overhauled, points to quay.io/konveyor org now
Downstream will replace the image value [here](https://github.com/openshift/oadp-operator/pull/1997/files#diff-ce0d597a9e5f20a8fe61c3ba5e6185a6b90cfef2c00bb5b5ef3e97b15b79f480R91-R92)